feat(Kommander): project addon catalog (#2776)
mesosphere/dcos-docs-site: pages/ksphere/kommander/latest/projects/index.md

## Platform Services

> Platform Services provide a catalog of applications that can be deployed across all project clusters. See [Addon Catalog](/ksphere/kommander/latest/projects/addon-catalog) for more information

Reviewer: ```suggestion
Platform Services provide a catalog of applications that can be deployed across all project clusters. See [Addon Catalog](/ksphere/kommander/latest/projects/addon-catalog) for more information.
```

> Platform Services provide a catalog of applications that can be deployed across all project clusters. See [Addon Catalog](/ksphere/kommander/latest/projects/addon-catalog) for more information

Reviewer 2: ```suggestion
Platform Services provide a catalog of applications you can deploy across all project clusters. See [Addon Catalog](/ksphere/kommander/latest/projects/addon-catalog) for more information
```
